appcircle_firebase_dsym_upload/1.0.1/component.yaml

```diff
@@ -0,0 +1,35 @@
+platform: iOS
+buildPlatform:
+displayName: Firebase Upload dSYM
+description: "Upload debug symbols to Firebase"
+docsUrl: https://docs.appcircle.io/workflows/common-workflow-steps#firebase-upload-dsym
+requiredComponents: "appcircle_ios_build_sign"
+precedingComponents: 
```

Comment:
Other build steps that give ipa output should also be added here.

Reply:
dSYM file generate only archived application in appcircle_ios_sign. So other steps does not generate dsym file.